.github/main.yml

```diff
@@ -0,0 +1,51 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  CI_PIPELINE:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Setup Go environment
+          uses: actions/setup-go@v3.0.0
+          with:
+            # The Go version to download (if necessary) and use. Supports semver spec and ranges.
+            go-version: 1.18
+            # Set this option to true if you want the action to always check for the latest available version that satisfies the version spec
+            check-latest: true
+
+      # Setup tools
+      - name: Setup tools
+        run: |
+          pip install pre-commit
+          pre-commit install
+
+      # Check for pre-commit updates if it is a renovate PR
+      - name: Pre-commit sweeper
+        run: module-assets/ci/pre-commit-sweeper.sh
+
+      # run pre-commit against all files
+      - name: Pre-commit
+        run: pre-commit run --all-files
+
+      # Run tests
+      - name: Run Tests
+        run: make run-tests
```

Comment:
Wrong make command. Infact you won't be able to use a current make command on public github.com. So go with `go test ./... -count=1 -v -timeout 5m`

Reply:
Fixed should work now